### ISSUE TITLE
Organize head development branches

### ISSUE THREAD

**Issue by jbohren** [ros-controls/ros_control#13]
Right now there are seemingly three development branches: `master`, `fuerte`, and `groovy`. Since this package hasn't been catkinized, should there be any distinction between these three?

**Comment by wmeeusse** [comment]
These branches are created by the release script; releases into fuerte and groovy were identical up till now. The only development branch we've been using is 'master'. But, in changeset c3b7f4db606dab68d6163c47ea0650aa46b76bd2 we changed the API, which is something we can't release any more into Fuerte, so from now on fuerte and groovy releases are going to be different. I created a new 'fuerte-dev' branch, but I imagine that from now on we'll only do releases into groovy, unless someone really cares about using ros_control with the old API in fuerte. We'll keep 'master' as the development branch for groovy as long as we can.
